.doc_gen/metadata/bedrock-runtime_metadata.yaml

```diff
@@ -189,6 +189,22 @@ bedrock-runtime_Scenario_ToolUse:
   synopsis: "build a typical interaction between an application, a generative AI model, and connected tools or APIs to mediate interactions between the AI and the outside world. It uses the example of connecting an external weather API to the AI model so it can provide real-time weather information based on user input."
   category: Scenarios
   languages:
+    Java:
```

Comment:
There are 3 metadata keys listed in the specification, looks like this is missing 2 of them.

Reply:
Fixed - added to Nova section